CONFIGURATION.md

```diff
@@ -36,6 +36,19 @@ Your configuration overrides the default configuration, which can be found
   // Can be: 'icon', 'name', 'iconAndName'
   "showJobs": "icon",
 
+  // If entries in showJobsNameOnStatus, will only show names for job statuses matching any of the names given.
+  // Array can have entries of the following: 'created', 'pending', 'running', 'failed','success', 'canceled', 'skipped' or 'manual'.
+  // See https://docs.gitlab.com/ee/api/jobs.html#list-project-jobs and available scopes.
+  // Example below is to only show job names for non-success jobs.
```

Comment:
@timoschwarzer We can reword this , as this is our use case, to only show job names on non-successful jobs. If a job is green, we don't really care what the success is.

Reply:
Feel free to use my proposed text :)